test/unit/development/version–bump-test.js

```diff
@@ -1,46 +0,0 @@
-import assert from 'assert'
-import versionBump from '../../../development/version-bump'
```

Comment:
`version-bump` doesn't seem to have been deleted. It looks like it should be.

Reply:
Whoops, fixed